prime-router/src/main/kotlin/azure/FHIRFlowFunctions.kt

```diff
@@ -16,6 +26,38 @@ const val fhirQueueName = "fhir-raw-received"
  * Process will work through all the reports waiting in the 'process' queue
  */
 class FHIRFlowFunctions : Logging {
+    /**
+     * An azure function for processing an HL7 message into of FHIR
+     */
+    @FunctionName("translate-hl7-fhir")
+    @StorageAccount("AzureWebJobsStorage")
+    fun receive(
+        @HttpTrigger(
+            name = "translate",
+            methods = [HttpMethod.POST],
+            authLevel = AuthorizationLevel.ANONYMOUS,
```

Comment:
This is currently an open endpoint. Should it be limited it by function key or some other auth method?